server/utils.ts

```diff
@@ -22,3 +22,13 @@ export const getChartWidthWithSize = (size: string) => {
     return 600;
   }
 };
+
+export async function getBase64Image(url: string): Promise<string> {
+  const fetch = (...args) =>
+    import("node-fetch").then(({ default: fetch }) => fetch(...args));
```

Comment:
I'm curious why dynamic imports are used here 🤔️. Does import `fetch` normally cause any problems?

Reply:
It's there because this method was declared under `packages/xy-chart/utils` before. I was using it to handle both browser side and server side so `fetch` would be imported only when on server. OK to remove it now, I'll do this.

Reply:
Updated.